extra.d.ts

```diff
@@ -0,0 +1,253 @@
+// *********************************************************************************************
+// This file is manually-edited by diffing against an autogenerated file. See README.md.
```

Comment:
I assume this is here to be pasted into the combined file, but it's confusing because extra.d.ts is meant to be all manually written right? Maybe move this comment into a string in the script?